### PR TITLE
Renamed macros to field_templates

### PR DESCRIPTION
**Pull request by shaharyarn**

### CODE REVIEW

File: marine/simple_marine.py [hunk 1]
```diff
@@ -103,15 +100,16 @@ def validate_display_filter(display_filter: str) -> bool:
 
 
 def validate_fields(
-    fields: List[str],
-    macros: Optional[Dict[str, List[str]]] = None,
+    fields: List[str], field_templates: Optional[Dict[str, List[str]]] = None
```
Comment (tomlegkov) [inline marine/simple_marine.py:103]: See my comment on formatting

File: marine/simple_marine.py [hunk 2]
```diff
@@ -103,15 +100,16 @@ def validate_display_filter(display_filter: str) -> bool:
...
 ) -> bool:
     """
     Validates the given fields. Fields have the same name as specified for Wireshark.
     If you want to add a custom field, you need to have the required dissector in your Wireshark plugins folder.
-    Macros can be used to expand a field - Example macro format: {"macro.ip.src" : ["ip.src", "ipv6.src"]}.
+    Field_templates can be used to expand a field - Example field_template format: {"macro.ip.src" : ["ip.src", "ipv6.src"]}.
```
Comment (tomlegkov) [inline marine/simple_marine.py]: `Field templates` or `field_templates`

File: marine/marine.py [hunk 3]
```diff
@@ -238,26 +248,26 @@ def __del__(self):
         if getattr(self, "_marine", None):
             self._marine.destroy_marine()
 
-    def _expand_macros(
-        self, fields: List[str], macros: Optional[Dict[str, List[str]]]
+    def _expand_field_templates(
+        self, fields: List[str], field_templates: Optional[Dict[str, List[str]]]
```
Comment (tomlegkov) [inline marine/marine.py:252]: Maybe also need to run Black here? Or does it place multiple parameters in the same line?

File: marine/simple_marine.py [hunk 4]
```diff
@@ -35,22 +35,22 @@ def filter_packet(
...
-    macros: Optional[Dict[str, List[str]]] = None,
+    field_templates: Optional[Dict[str, List[str]]] = None,
     encapsulation_type: Optional[int] = None,
 ) -> Dict[str, Optional[str]]:
     """
     Parses the given fields from the packet. Fields have the same name as specified for Wireshark.
     If you want to add a custom field, you need to have the required dissector in your Wireshark plugins folder.
     Fields that are not available in the packet will be returned as "".
-    Macros can be used to expand a field - Example macro format: {"macro.ip.src" : ["ip.src", "ipv6.src"]}.
+    Field_templates can be used to expand a field - Example field_template format: {"macro.ip.src" : ["ip.src", "ipv6.src"]}.
```
Comment (tomlegkov) [inline marine/simple_marine.py]: `Field templates` or `field_templates`

**Review by tomlegkov — CHANGES_REQUESTED** [review]

**Review by yehonatanz — APPROVED** [review]
Approved up to @tomlegkov's comments

File: marine/marine.py (hunk 3 above)
Comment (shaharyarn) [reply marine/marine.py:252]: It is blacked. It does place the parameters in the same line. 
Same goes for the other formatting comment.

**Review by tomlegkov — APPROVED** [review]

**Pull request merged**